src/index.html

```diff
@@ -1,22 +1,100 @@
 <!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
+    <meta
+      charset="UTF-8"
+    />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, user-scalable=no"
     />
     <meta
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
-    />
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    >
+    <title>Moyo header</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="header__logo_url"
+        href="index.html"
+      >
+        <img
+          class="header__logo"
+          src="moyo_1.png"
+          alt="header_logo"
+        >
+      </a>
+      <!-- nav.nav>ul.nav__list>li*8.nav__li>a.nav__url -->
+      <nav class="nav">
+        <div class="active-underline"></div>
```

Comment:
try to implement this task without using the tag for underline 
```suggestion
```

Reply:
> try to implement this task without using the tag for underline

Made